### PR TITLE
More efficient prime check

### PR DESCRIPTION
**Pull request by raklaptudirm**
Increased `PrimeCheck` algorithm efficiency, making it 2x faster. Complexity remains `O(sqrt(n))`.
# Welcome to JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by raklaptudirm** [comment]
- [ ] Standard style update

File: Maths/PrimeCheck.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ const PrimeCheck = (n) => {
   // output: boolean
   if (n === 1) return false
   if (n === 0) return false
+  if (n % 2 === 0) return false
```
Comment (ruppysuppy) [inline Maths/PrimeCheck.js:14]: This will lead to 
```js
> PrimeCheck(2)
false
```

**Review by ruppysuppy — CHANGES_REQUESTED** [review]

File: Maths/PrimeCheck.js (hunk 1 above)
Comment (raklaptudirm) [reply Maths/PrimeCheck.js:14]: I somehow deleted the fork. Please see #596 for the update.

**Comment by raklaptudirm** [comment]
Updates at #596

**Pull request closed without merging**